pixi/src/ui/report/CoreWebVitalsReportView.js

```diff
@@ -134,7 +134,7 @@ class CoreWebVitalView {
   }
 
   reset() {
-    this.container.classList.remove(...Object.values(CATEGORIES));
+    this.container.classList.remove(...Object.keys(CATEGORIES));
```

Comment:
At one point removing multiple classes at once didn't work and worker-dom, I'm assuming this was fixed?